### DISCUSSION TITLE
posts/nixos-proxmox-lxc-setup

### DISCUSSION THREAD

**Discussion by giscus** [Jacob-Swanson/Jacob-Swanson.github.io#27] [Announcements]
# tech/2023/09/19/nixos-proxmox-lxc-setup

This guide will explain how to install NixOS in an LXC container in Proxmox.

https://www.jacob-swanson.com/tech/2023/09/19/nixos-proxmox-lxc-setup.html

**Comment by johntdavis84** [comment]
This is an awesome tutorial, and exactly the sort of thing I was looking for to set up a NixOS LXC to run docker in.

Are these instructions still valid in July 2024?

**Reply by jacob-swanson** [reply]
Yep, they're still valid. :)

**Comment by linwanxiaoyehua** [comment]
https://hydra.nixos.org/job/nixos/trunk-combined/nixos.proxmoxLXC.x86_64-linux
check this out, now ProxmoxLXC img is aviliable.
